common/constants.js

```diff
@@ -18,6 +18,8 @@ export const font = {
   medium: `'inter-medium', -apple-system, BlinkMacSystemFont, arial, sans-serif`,
   mono: `'mono', monaco, monospace`,
   monoBold: `'mono-bold', monaco, monospace`,
+  monoCode: `'firaCode-regular', mono, monospace`,
```

Comment:
Stylistically I would prefer `fira-code-regular`, I should have been more clear here, could you update this?